vspace/src/main/java/edu/asu/diging/vspace/web/staff/DeleteSpaceController.java

```diff
@@ -0,0 +1,28 @@
+package edu.asu.diging.vspace.web.staff;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.diging.vspace.core.services.ISpaceManager;
+
+@Controller
+public class DeleteSpaceController {
+    
+    @Autowired
+    private ISpaceManager spaceManager;
+
+    @RequestMapping(value = "/staff/space/{id}", method = RequestMethod.DELETE)
+    public ResponseEntity<?> deleteSpace(@PathVariable String id) {
+        try {
+            spaceManager.deleteSpaceById(id);
+        } catch(Exception exception) {
```

Comment:
what exception is caught here? never catch Exception if you can catch a more specific one.

Reply:
Especially if you don't log the exception. This could be any kind of exception and there would be no way for us to find out what actually went wrong.